modules/nodes-scheduler-node-selectors-pod.adoc

```diff
@@ -64,7 +64,7 @@ The web console lists the controlling object under `ownerReferences` in the pod
 
 * Use a `MachineSet` object to add labels to nodes managed by the machine set when a node is created:
 
-.. Run the following command to add a node selector to a `MachineSet` object:
+.. Run the following command to add a label to a `MachineSet` object:
```

Comment:
I think it's better to replace "a label" with "labels", for there are two pairs of <key, value>